cython/aabb.pyx

```diff
@@ -0,0 +1,59 @@
+from libcpp cimport bool
+
+
+cdef class AxisAlignedBoundingBox(Hittable):
```

Comment:
This class has already been implemented in Python.

Probably worth it to do a Cython implementation at some point though

Reply:
Actually we do have the Cython Implementation